Chinese/YingJeouMa.md

```diff
@@ -0,0 +1 @@
+成為某國總統，禍國殃民。
```

Comment:
It looks like this file has been added to a directory that already contained a "page" of the story.  Please create a new directory for each new page to help keep the repository organized.  See the README of this repository for more information.